src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -711,14 +711,20 @@ public static void validate(final boolean condition, final Supplier<String> msg)
      * @throws UserException if {@code file} is not a regular file or it cannot be read.
      * @return the same as the input {@code file}.
      */
-    public static File regularReadableUserFile(final File file) {
+    public static void canRead(final File file) {
```

Comment:
Is there a need for both a single-argument method along with one with varargs?  I'd just put the loop in here.  Be sure to fix up the javadoc accordingly.  Also, might want to add an additional null check in case only a single null argument is passed, as well as define behavior when zero arguments are passed.